temporal-sdk/src/main/java/io/temporal/internal/statemachines/CancellableCommand.java

```diff
@@ -67,7 +67,7 @@ public void handleCommand(CommandType commandType) {
   public WorkflowStateMachines.HandleEventStatus handleEvent(
       HistoryEvent event, boolean hasNextEvent) {
     if (canceled) {
-      return WorkflowStateMachines.HandleEventStatus.NON_MATCHING_EVENT;
+      return WorkflowStateMachines.HandleEventStatus.COMMAND_CANCELLED;
     }
```

Comment:
Should this change be in this branch?

Reply:
No, unrelated, and needed more for clarity and documentation. This change shouldn't change anything.
It's just incorrect to say that the event is not matching when the command is canceled and we never even checked it.
Also, with the current code this path should never actually happen.